docs/create-sections.html

```diff
@@ -3,5 +3,28 @@
 title: Create Sections
 ---
 
-<h1 class="mt-5">Create Sections Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Create Sections</h1>
+<p>
+  Some courses benefit from additional sections to organize students/participants for a variety of reasons. 
+  Course Manager makes new section creation quick and easy by uploading a simple CSV file.
+</p>
+<p>
+  <b>Tip: </b>Authorized users can add sections on the fly when adding U-M or non-U-M users with this tool. 
+  See the help documentation for these features for more information.
+</p>
+<h2 class="h5">Who Can Create Sections</h2>
+<p>Section management activities in this tool are limited to administrator roles:</p>
+<ul>
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```

Comment:
Same deal, I'd make this a `p` tag outside the list.

Reply:
@melindakraft prefers this way. I will keep it in the list. Thanks!